### ISSUE TITLE
Electricity load timeseries for 8784 hours

### ISSUE THREAD

**Issue by ClaraBuettner** [openego/eGon-data#593]
In` grid.egon_etrago_load_timeseries `all 'AC' loads have more p_set values than expected:
```
SELECT * FROM grid.egon_etrago_load
WHERE load_id IN
(SELECT load_id
FROM grid.egon_etrago_load_timeseries
WHERE cardinality(p_set) != 8760);
```
This seems to be caused by the household profiles, the input table `demand.egon_etrago_electricity_households` already has values for 8784 hours:

```
SELECT cardinality(p_set)
FROM demand.egon_etrago_electricity_households
WHERE cardinality(p_set) != 8760;
```
I used the data from the latest CI-run. 
@nesnoj @nailend Do you have an idea why this happened?

**Comment by nesnoj** [comment]
Yes, `len(pd.read_hdf('household_electricity_demand_profiles_2400.hdf5'))` reveals that the original timeseries holds data for 8784h, apparently they chose a leap year :-/.
Shall we remove February 29th?

**Comment by ClaraBuettner** [comment]
> Yes, `len(pd.read_hdf('household_electricity_demand_profiles_2400.hdf5'))` reveals that the original timeseries holds data for 8784h, apparently they chose a leap year :-/. Shall we remove February 29th?

When we delete February 29th we mix up the weekdays. But that happens anyhow when the data was created for another year... Do you know if the household profiles are very different during a week?

**Comment by nesnoj** [comment]
> What year are the heat timeseries generated for, 2011?

In your script [HTS.py](https://github.com/openego/eGon-data/blob/bc4aec0148ac02b6f8d278accf39625e6dcd5bfc/src/egon/data/datasets/heat_demand_timeseries/HTS.py) you use 2011 but is this for indexing purposes only or does it incorporate the weekly patterns too?

**Comment by nesnoj** [comment]
I've prepared a fix in #597.
@ClaraBuettner If you agree with my underlying assumptions above, it could be deployed pretty soonish. Currently executing in test mode..

**Comment by ClaraBuettner** [comment]
Removing the first day seems to be the best solution for me. The holidays will not be the same, but that's less critical. 
We are not dividing heat profiles into different weekdays because they don't depend that much on the weekday since (space) heating demand is less dependent on the presence of persons. 
But the CTS and industry electricity profiles depend on the weekday, especially some industrial profiles differ at the weekend. So I was a bit concerned to have an inconsistent electrical load profile when we consider different weekdays.
